core/components/atoms/popperWrapper/PopperWrapper.tsx

```diff
@@ -291,14 +324,43 @@ export class PopperWrapper extends React.Component<PopperWrapperProps, PopperWra
     );
   }
 
+  getRandomString() {
+    const randomChars = 'abcdefghijklmnopqrstuvwxyz';
+    let result = '';
+    for (let i = 0; i < 5; i++) {
+      result += randomChars.charAt(Math.floor(Math.random() * randomChars.length));
+    }
+    return result;
+  }
+
```

Comment:
you may use Math.random().toString(36).substr(2, 5); for calculating  random string